intg/src/main/java/org/apache/atlas/model/tasks/TaskSearchParams.java

```diff
@@ -67,4 +67,14 @@ private Map mapOf(String key, Object value) {
 
         return ret;
     }
+
+    public Map getDSLGuid(String taskGuid){
```

Comment:
No need to add this here 
We are already doing the same in TaskRegistry.java, see the following method

_public List<AtlasTask> getByIdsES(List<String> guids) throws AtlasBaseException {_

I suggest moving this logic to a separate method which will return TaskSearchParams to use at both places

Reply:
If needed we can move this to AtlasTaskService.java

Reply:
getByIdsES already uses getTasks function inside AtlasTaskService. So, if we try to use TaskRegistry inside AtlasTaskService there will be a cyclic dependency.